src/cloud/components/Views/Table/TableViewContentManager.tsx

```diff
@@ -146,8 +158,16 @@ const TableViewContentManager = ({
         title: getDocTitle(doc, 'untitled'),
       }
     })
-    return sortByAttributeAsc('title', filteredDocs)
-  }, [documents])
+    switch (order) {
+      case 'Title A-Z':
+        return sortByAttributeAsc('title', filteredDocs)
+      case 'Title Z-A':
+        return sortByAttributeDesc('title', filteredDocs)
+      case 'Latest Updated':
+      default:
+        return sortByAttributeDesc('updatedAt', filteredDocs)
+    }
+  }, [order, documents])
 
   const orderedFolders = useMemo(() => {
```

Comment:
Folders need to be reordered as well imo as we are keeping the same logic for the whole page now